### PR TITLE
Add Interface Entity

### PR DESCRIPTION
**Pull request by Cztery**
*New Entity - Interface.
*Additional debug functionalities.

### CODE REVIEW

File: src/Interface.cpp [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+            Attribute::ValueTypes::STRING, NL80211_CMD_GET_INTERFACE,
+            NL80211_CMD_SET_INTERFACE) {
+  set_identifier(id);
+}
+
+const uint32_t &Interface::get_identifier() const { return index_.get_value(); }
+
+void Interface::set_identifier(const uint32_t &id) { index_.set_value(id); }
+
+}  // namespace wiphynlcontrol
```
Comment (piotrkoziar) [inline src/Interface.cpp]: nit: newline at the end of file

File: src/Interface.h [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+ public:
+  // Returns identifier. Overloads abstract method from Entity.
+  const uint32_t &get_identifier() const;
+  // Overloads abstract method from Entity.
+  void set_identifier(const uint32_t &id);
+};
+
+}  // namespace wiphynlcontrol
+#endif
```
Comment (piotrkoziar) [inline src/Interface.h]: nit: newline

File: src/Property.cpp [hunk 3]
```diff
@@ -31,9 +31,11 @@ void Property<T>::set_value(T val) {
 
 template <typename T>
 const T &Property<T>::get() {
+  auto tmp_attr =
```
Comment (piotrkoziar) [inline src/Property.cpp]: It seems like an unnecessary variable!

**Comment by piotrkoziar** [comment]
Great job, @Cztery !

File: src/Interface.cpp (hunk 1 above)
Comment (Cztery) [reply src/Interface.cpp]: done 5c904ed3bd1e4de5104225562791891596d6803c

File: src/Interface.h (hunk 2 above)
Comment (Cztery) [reply src/Interface.h]: done 5c904ed3bd1e4de5104225562791891596d6803c

File: src/Property.cpp (hunk 3 above)
Comment (Cztery) [reply src/Property.cpp]: done 5c904ed3bd1e4de5104225562791891596d6803c

**Pull request merged**